app/services/force/soql/preference_service.rb

```diff
@@ -52,12 +68,14 @@ def app_preferences_for_listing_query(opts)
       # complexity of the query we are dynamically adding filters
       # within the WHERE clause with the helper function above. The
       # Preference_All_Name and Preference_All_Lottery_Rank fields
-      # were added to bring all prefernces into the same query.
+      # were added to bring all preferences into the same query.
       filters = buildAppPreferencesFilters(opts)
+      search = opts[:search] ? buildAppPreferencesSearch(opts[:search]) : nil
```

Comment:
if `opts[:search]` is an empty string, this will set `search` to nil, right?

Reply:
yeah that was the idea